.travis.yml

```diff
@@ -68,6 +68,8 @@ script:
   - make integration-test-disable-wallet-api
   # Disable seed api test
   - make integration-test-disable-seed-api
+  # Test api/v2
+  - make integration-test-stable-v2
```

Comment:
Shouldn't these be included side-by-side with v1 API requests ?

Reply:
Yes, they can be in the same test file and same script but I thought they would better in a separate file.
OK. I will put them toguether.

Reply:
I did that. All stable integration tests are together.